docs/libs/creating-lib.md

```diff
@@ -207,7 +230,7 @@ Add the following scripts:
 "release:myaccount:with-changelog": "cd feature-libs/my-account && release-it && cd ../..",
 ```
 
-And replace `myaccount` and `my-account` instances with the name of yours lib.
+And replace `asm` instances with the name of yours lib.
```

Comment:
Shouldn't in this case example be adjusted for the asm lib?

Reply:
Good catch.  Fixed.
The text got messed up at some point when resolving a merge conflict.